woocommerce/woocommerce-android
# [Scan to Update Inventory] Hide Scan Barcode Menu Item if store has Woo Square Plugin Active

### NOTE: [This FluxC PR ](https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2912)needs to be reviewed and merged first

Closes: #10266 


### Description

context: pdfdoF-47p-p2#comment-5261

This PR adds the logic to hide the Scan Barcode icon from the products list if the Woo Square plugin is active. 


### Testing instructions


- Use a store where the Woo Square plugin is installed and active
- Go to the product list
- Make sure the Scan Barcode icon in the menu bar is not being displayed
- Use a store without Woo Square
- Go to the products list
- Make sure the Scan Barcode icon is displayed in the menu.

### Images/gif


<img src="https://github.com/woocommerce/woocommerce-android/assets/30724184/4042beaa-7320-430d-83c8-9d92f1678a1a" width=250>
